content/post/blacklivesmatter.md

```diff
@@ -0,0 +1,66 @@
+- Been learning to take a step back
+  - Had feedback that I get too involved with everything
+  - Told I keep trying to solve problems
+  - I wanted to hear other peoples voices to understand what was going on
+  - I've never seen colour, race, religion, gender, sexuality as an issue
+  - The understanding my personal opinion solves the problem
+  - This post has started a lot with points about me
+  - Still trying to figure out if it was wrong of me to wait to try to understand what has and is going on
+- This isn't an American problem
+  - Easy to think that this is someone elses problem
+  - Harder to look and realise that what you thought was equal isn't
+  - Even harder to want to change the normal to make it fair
+  - Do we have these issues in the UK and in Wales
+- Reached out to friends who would be deemed to be in a minority
+  - Listening to others, I now have a better understanding that this isn't about now, this is about the past and the future
+  - Have found out that people expect to be treated differently
+  - Was it wrong that I thought it'd be racist of me to presume that my friend would get pulled over for driving a nice car in Cardiff?
+  - Was it wrong for me to think I was being racist for being yet another privileged friend to ring and ask them how they're doing and how it affects them?
+- How do we solve this?
+  - Don't know how we solve this, but I know we can make this better each and every day
+  - Promote diversity in your workspace, at home and everywhere you look
+  - Offer to help someone, wether its with a pull request, mentoring, or putting someone else in front of you to give them the gift that we've been given for so long that we no longer see it as a gift as much as a right
+  - Please take the opportunity to add to this blog post to share what we can do to make a change
+- Now I have your attention
+  - Using the pseudonym M until I have permission to share their details
+  - Before COVID I trialed a mentoring scheme to help find M his first Data Science role
+    - The schemes focus is to improve diversity within the technology industry
+  - M is a final year student in Cardiff whilst also working as a Care Worker
+  - M and I paired up at conference workshops
+  - M was donated a laptop from a local business in Cardiff
+  - M had passed the initial steps for his first interview
+  - Lockdown happened
+  - M managed to get to the final stage of the interview and sadly it didn't work out
+  - M is still looking for his first Data Science role
+  - M is still working as a Care Assistant and has supported the elderly in Cardiff during COVID
+  - If you can help find a role for M then we're starting to make a difference
+  - Please [contact me](mailto:lewis@cloudnativewales.io) and I'll direct you to M
+    - Awaiting confirmation from M that I can share their contact details so you can message them directly
+- Wales builds on its communities
```

Comment:
I'd suggest removing this section about Welshness and allyship. It would also help the blog post to avoid becoming overly long - there are a lot of words being published and blogged about BLM. We need to be careful that we're not adding too much of our own noise, but instead show our support and help community members listen to black voices and think about measurable actions that they can take to confront systemic racism

Reply:
I agree, after thinking about it since writing the draft yesterday I feel that this is too open ended and every point that you said.